dyno-queues-redis/src/main/java/com/netflix/dyno/queues/redis/RedisDynoQueue.java

```diff
@@ -99,13 +101,18 @@ public RedisDynoQueue(String redisKeyPrefix, String queueName, Set<String> allSh
 	}
 
 	public RedisDynoQueue(Clock clock, String redisKeyPrefix, String queueName, Set<String> allShards, String shardName, int unackScheduleInMS) {
+		this(clock, redisKeyPrefix, queueName, allShards, shardName, unackScheduleInMS, null);
```

Comment:
it may better to have a default shardingStrategy; without one, push() will throw an Exception.